android/anc/src/main/java/org/smartregister/fhircore/anc/ui/family/form/RemoveFamilyQuestionnaireActivity.kt

```diff
@@ -0,0 +1,126 @@
+package org.smartregister.fhircore.anc.ui.family.form
+
+import android.app.AlertDialog
+import android.content.DialogInterface
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Button
+import androidx.activity.viewModels
+import dagger.hilt.android.AndroidEntryPoint
+import org.hl7.fhir.r4.model.QuestionnaireResponse
+import org.smartregister.fhircore.anc.ui.details.PatientDetailsActivity
+import org.smartregister.fhircore.anc.ui.family.details.FamilyDetailsActivity
+import org.smartregister.fhircore.anc.util.othersEligibleForHead
+import org.smartregister.fhircore.engine.R
+import org.smartregister.fhircore.engine.ui.base.AlertDialogListItem
+import org.smartregister.fhircore.engine.ui.base.AlertDialogue
+import org.smartregister.fhircore.engine.ui.base.AlertDialogue.getSingleChoiceSelectedKey
+import org.smartregister.fhircore.engine.ui.questionnaire.QuestionnaireActivity
+import org.smartregister.fhircore.engine.util.extension.showToast
+
+@AndroidEntryPoint
+class RemoveFamilyQuestionnaireActivity : QuestionnaireActivity() {
+    lateinit var saveBtn: Button
+    private lateinit var familyId: String
+
+    override val questionnaireViewModel by viewModels<RemoveFamilyQuestionnaireViewModel>()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setupUI()
+        loadData()
+    }
+
+    private fun loadData() {
+        familyId = intent.extras?.getString(QUESTIONNAIRE_ARG_PATIENT_KEY) ?: ""
```

Comment:
@ellykits Same question I have earlier, can we create an object for the family & patient. We can then pass this object to the other classes any time we need to.  Right now we are passing single attributes (which is okay) but will it be too many intent etxras at some point?